ci/concourse/azure-template-updater.yml

```diff
@@ -52,30 +102,64 @@ jobs:
               repository: node
               tag: 14-alpine
           inputs:
-            - name: git-repo-fortigate-autoscale-azure
+            - name: git-repo-fortigate-autoscale-azure-updated
             - name: resource-azure-list-fortigate-images-byol
             - name: resource-azure-list-fortigate-images-payg
             - name: resource-azure-list-fortianalyzer-images-byol
           outputs:
             - name: git-repo-fortigate-autoscale-azure-updated
           run:
-            path: 'sh'
+            path: node
+            args:
+              # first arg is the script to run
+              # working directory will be resolved into 'git-repo-fortigate-autoscale-azure-updated'
+              - git-repo-fortigate-autoscale-azure-updated/build/scripts/template-updater.js
```

Comment:
this first arg can be moved to `path:` if you add the node shebang to your ts file?

`#!/usr/bin/env node`